BUILD.md

```diff
@@ -20,7 +20,23 @@ poetry install
 When poetry is done installing all dependencies you can start using the tool.
 
 ```shell
-poetry run python -m tad
+poetry run python -m uvicorn tad.main:app
```

Comment:
This is the 'old' startup, can you update it?

Reply:
Done